### PR TITLE
Make Premium page more responsive

### PR DESCRIPTION
**Pull request by dem4ron**
#### Logged out:

https://github.com/exercism/website/assets/66035744/03b50819-5279-413a-92c5-62eedc85a032

#### Logged in:

https://github.com/exercism/website/assets/66035744/539b8c6e-24e8-49e6-9fe4-e05004f6ab07

#### Logged in && Premium:

https://github.com/exercism/website/assets/66035744/29851499-e971-4369-9d53-ea4dccefe830

### CODE REVIEW

File: app/helpers/view_components/site_header.rb [hunk 1]
```diff
@@ -23,8 +23,8 @@ def announcement_bar
         tag.div(class: "lg-container") do
           graphical_icon(:premium, css_class: 'h-[16px] w-[16px] mr-12') +
             tag.span("Exercism Premium has launched.") +
-            tag.strong("Supercharge your experience!") +
-            tag.div("Check it out", class: 'btn-primary btn-xs ml-4')
+            tag.strong("Click here to learn more.") +
+            tag.div("Check it out", class: 'btn-primary btn-xs ml-4 hidden md:block')
```
Comment (iHiD) [inline app/helpers/view_components/site_header.rb]: Why have we lost "Supercharge your experience"?

File: app/views/premium/external.html.haml [hunk 2]
```diff
@@ -65,6 +65,6 @@
         .price-card.border-2.border-purpleDarkened
           %h2.text-h2 Become an Insider
           .price $499
-          %p.text-p-base.mb-32 Enjoy full lifetime access to both Exercism Premium and Exercism Insiders, giving you behind-the-scenes access to videos and exclusive team community spaces.
+          %p.text-p-base Enjoy full lifetime access to both Exercism Premium and Exercism Insiders, giving you behind-the-scenes access to videos and exclusive team community spaces.
```
Comment (iHiD) [inline app/views/premium/external.html.haml]: Why has this lost the bottom margin?

Comment (iHiD) [reply app/views/premium/external.html.haml]: (Losing the margin has resulted in this, so I think you want to revert this, or change L70 to be `.mt-32= render ReactComponents::Premium::PriceOption.new(:lifetime)` maybe)

Screenshot 2023-06-19 at 17 19 19

File: app/helpers/view_components/site_header.rb (hunk 1 above)
Comment (dem4ron) [reply app/helpers/view_components/site_header.rb]: Sorry, this is a wrong version. It got lost in branch juggling. :(

**Pull request merged**